### PR TITLE
Link to Narwhal docs throughout Learn section and in FAQ

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: doc/src/learn/how-sui-works.md [hunk 1]
```diff
@@ -52,11 +52,11 @@ Sui validators agree on and execute transactions in parallel with high throughpu
 
 This section is written for a technical audience wishing to gain more insight about how Sui achieves its main performance and security objectives.
 
-Sui assumes the typical blockchain transaction is a user-to-user transfer or asset manipulation and optimizes for that scenario. As a result, Sui distinguishes between two types of assets (i) owned objects that can only be modified by its specific owner, and (ii) shared objects that have no specific owners and can be modified by more than one user. This distinction allows a design that forgoes consensus for simple transactions involving only owned objects to achieve very low latency.
+Sui assumes the typical blockchain transaction is a user-to-user transfer or asset manipulation and optimizes for that scenario. As a result, Sui distinguishes between two types of assets (i) owned objects that can only be modified by its specific owner, and (ii) shared objects that have no specific owners and can be modified by more than one user. This distinction allows a design that forgoes [consensus](architecture/consensus.md) for simple transactions involving only owned objects to achieve very low latency.
```
Comment (randall-Mysten) [inline doc/src/learn/how-sui-works.md:55]:
```suggestion
Sui assumes the typical blockchain transaction is a user-to-user transfer or asset manipulation and optimizes for that scenario. As a result, Sui distinguishes between two types of assets (i) owned objects that can be modified only by their specific owner, and (ii) shared objects that have no specific owners and can be modified by more than one user. This distinction allows for a design that achieves very low latency by forgoing [consensus](architecture/consensus.md) for simple transactions involving only owned objects.
```

File: doc/src/learn/sui-compared.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ A lot of transactions do not have complex interdependencies with other, arbitrar
...
 
-This doesn't mean that Sui as a platform never orders transactions with respect to each other, or that it allows owners to only affect their owned microcosm of objects. Sui will also process transactions that have an effect on some shared state, in a rigorous, consensus-ordered manner. They're just not the default use case.
+This doesn't mean that Sui as a platform never orders transactions with respect to each other, or that it allows owners to only affect their owned microcosm of objects. Sui will also process transactions that have an effect on some shared state, in a rigorous, consensus-ordered manner. They're just not the default use case. See the [State-of-the-art consensus](#state-of-the-art-consensus) section for details on the [Narwhal and Tusk consensus engine](architecture/consensus.md).
```
Comment (randall-Mysten) [inline doc/src/learn/sui-compared.md:27]:
```suggestion
This doesn't mean that Sui as a platform never orders transactions with respect to each other, or that it allows owners to affect only their owned microcosm of objects. Sui also processes transactions that have an effect on some shared state in a rigorous, consensus-ordered manner. They're just not the default use case. See the [State-of-the-art consensus](#state-of-the-art-consensus) section for details on the [Narwhal and Tusk consensus engine](architecture/consensus.md).
```

**Review by randall-Mysten — APPROVED** [review]

File: doc/src/learn/how-sui-works.md (hunk 1 above)
Comment (Clay-Mysten) [reply doc/src/learn/how-sui-works.md:55]: Thanks, Randall. Great edits!

File: doc/src/learn/sui-compared.md (hunk 2 above)
Comment (Clay-Mysten) [reply doc/src/learn/sui-compared.md:27]: Good stuff!

**Review by huitseeker — APPROVED** [review]
Thanks a bunch! Just one comment

**Pull request merged**